composer.json

```diff
@@ -13,27 +19,47 @@
     }
   ],
   "scripts": {
-    "test:unit": "./vendor/bin/phpunit --color",
+    "test": "./vendor/bin/phpunit --color",
+    "test:unit": "./vendor/bin/phpunit --color --testsuite Unit",
+    "test:integration": "./vendor/bin/phpunit --color --testsuite Integration",
     "test:typing": "./vendor/bin/phpstan analyse --level max src",
     "test:syntax": "./vendor/bin/php-cs-fixer fix ./src --dry-run --stop-on-violation --using-cache=no"
   },
   "require": {
     "php": "^7.1",
     "ext-json": "*",
+    "ext-curl": "*",
     "ext-gmp": "*",
-    "lib-openssl": "*",
-    "guzzlehttp/guzzle": "^6.2",
+    "ext-openssl": "*",
+    "ext-mbstring": "*",
     "web-token/jwt-signature": "^1.0",
-    "web-token/jwt-key-mgmt": "^1.0"
+    "web-token/jwt-key-mgmt": "^1.0",
```

Comment:
All web-token deps could be bumped to 2.0.
You can let it as is at the moment. I will change that after this PR got merged (requires some modifications).